docs/undocumented.json

```diff
@@ -2,5 +2,5 @@
   "warnings": [
 
   ],
-  "source_directory": "/Users/smartcar/Desktop/ios-sdk"
+  "source_directory": "/Users/allisonchen/smartcar/ios-sdk"
```

Comment:
does this file need to be commited?

Reply:
oh hmm probably not! will remove + .gitignore